### PR TITLE
Preparing database

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by hoangminhuet** [comment]
ready

**Comment by hoangminhuet** [comment]
ready

**Comment by hainh-2132** [comment]
1 PR - 1 commit

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -22,7 +22,6 @@ public function up()
             $table->string('address');
             $table->string('phone');
             $table->integer('role_id')->unsigned();
-            $table->foreign('role_id')->references('id')->on('roles')->onDelete('cascade');
```
Comment (hainh-2132) [inline database/migrations/2014_10_12_000000_create_users_table.php:25]: không sửa file migration

File: database/migrations/2020_06_09_065919_create_orders_table.php [hunk 2]
```diff
@@ -16,7 +16,6 @@ public function up()
         Schema::create('orders', function (Blueprint $table) {
             $table->increments('id');
             $table->integer('user_id')->unsigned();
-            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
```
Comment (hainh-2132) [inline database/migrations/2020_06_09_065919_create_orders_table.php:19]: không sửa file migration

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 1 above)
Comment (hoangminhuet) [reply database/migrations/2014_10_12_000000_create_users_table.php:25]: ok anh

File: database/migrations/2020_06_09_065919_create_orders_table.php (hunk 2 above)
Comment (hoangminhuet) [reply database/migrations/2020_06_09_065919_create_orders_table.php:19]: ok anh

**Comment by hainh-0963** [comment]
1 PR - 1 commit

**Pull request merged**